lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -333,48 +346,19 @@ public static function get_merged_data( $origin = 'custom' ) {
 		return $result;
 	}
 
-	/**
-	 * Registers a Custom Post Type to store the user's origin config.
-	 */
-	public static function register_user_custom_post_type() {
```

Comment:
This has been extracted to `lib/compat/wordpress-5.9/register-global-styles-cpt.php`